src/tensors/factorizations.jl

```diff
@@ -36,6 +36,16 @@ function tsvd(t::AbstractTensorMap, (p₁, p₂)::Index2Tuple; kwargs...)
     return tsvd!(permute(t, (p₁, p₂); copy=true); kwargs...)
 end
 
+LinearAlgebra.svdvals(t::AbstractTensorMap) = LinearAlgebra.svdvals!(copy(t))
+function LinearAlgebra.svdvals!(t::AbstractTensorMap)
+    return SectorDict(c => LinearAlgebra.svdvals!(b) for (c, b) in blocks(t))
+end
+
+# TODO: decide if we want to keep these specializations:
```

Comment:
In line of the plans to stop the special casing of `TrivialTensorMap`, I would support to remove these specialisations.

Reply:
I definitely can, but in order to keep this consistent with `diag` and `diagm`, this would then also return a dictionary object for `TrivialTensorMap`. I don't have a big opinion on this, as I usually write code for generic symmetries and have to deal with these cases separately anyways, but I just want to mention that this is a bit connected here.